src/server/pachyderm_test.go

```diff
@@ -3114,7 +3112,7 @@ func TestStandby(t *testing.T) {
 			require.NoError(t, err)
 		}
 
-		require.NoErrorWithinTRetry(t, time.Second*30, func() error {
+		require.NoErrorWithinTRetry(t, time.Second*60, func() error {
```

Comment:
it seems that  getting all pipelines up in standby mode is taking longer than it did in the past. Should we investigate this as a performance regression, or does this seem acceptable? @brycemcanally @Tryneus

Reply:
This seems like something we should file an issue for and investigate - I think we're going to be putting a bigger emphasis on tracking down performance regressions in the next few weeks.